src/components/portrait/portrait.d.ts

```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
+
+export type PortraitShapes = "circle" | "square";
+
+export type PortraitSizes = "XS" | "S" | "M" | "ML" | "L" | "XL" | "XXL";
+
+export interface PortraitBaseProps {
+  /** The size of the Portrait. */
+  size?: PortraitSizes;
+  /** The `alt` HTML string. */
+  alt?: string;
+  /** The shape of the Portrait. */
+  shape?: PortraitShapes;
+  /** The initials to render in the Portrait. */
+  initials?: string;
+  /** Use a dark background. */
+  darkBackground?: boolean;
+  /** Prop for `onClick` events. */
+  onClick?: (ev: React.MouseEvent<HTMLElement>) => void;
+  /** The message to be displayed within the tooltip */
+  tooltipMessage?: React.ReactNode;
+  /** The id attribute to use for the tooltip */
+  tooltipId?: string;
+  /** Whether to to show the Tooltip */
+  tooltipIsVisible?: boolean;
+  /** Sets position of the tooltip */
+  tooltipPosition?: "top" | "bottom" | "left" | "right";
+  /** Defines the message type */
+  tooltipType?: string;
+  /** Defines the size of the tooltip content */
+  tooltipSize?: "medium" | "large";
+  /** Override background color of the Tooltip, provide any color from palette or any valid css color value. */
+  tooltipBgColor?: string;
+  /** Override font color of the Tooltip, provide any color from palette or any valid css color value. */
+  tooltipFontColor?: string;
+}
+
+export interface PortraitWithGravatar extends PortraitBaseProps {
+  /** An email address registered with Gravatar. */
+  gravatar?: string;
+}
```

Comment:
you have to add 
`src?: never;`
so the TypeScript will throw an error when the `src` prop is defined

Reply:
Applied